sample/shared_code/src/commonMain/kotlin/com/freeletics/flowredux/sample/shared/PaginationStateMachine.kt

```diff
@@ -73,7 +73,14 @@ data class ShowContentAndLoadingNextPageErrorPaginationState(
     internal override val canLoadNextPage: Boolean
 ) : ContainsContentPaginationState()
 
-internal class InternalPaginationStateMachine(
+/**
+ * It's callend `Internal` because it is note meant to be accessed publicly as it exposes coroutines
+ * Flow and suspending function to dispatch.
+ *
+ * Instead the "wrappter class" [PaginationStateMachine] should be used which hides `Flow` etc.
+ * but uses traditional "callbacks". That way it is easier to use on iOS.
+ */
+class InternalPaginationStateMachine(
```

Comment:
The idea was to only expose publicly `PaginationStateMachine` and not this class `InternalPaginationStateMachine` because `PaginationStateMachine` provides traditional callback based listener so that from iOS you don't need to bridge `Flow` and `suspend function`. 

but compose actually should access directly a `FlowReduxStateMachine` to be able to use extension functions like `stateAndDispatch()` on it. Therefore made it public.